templates/service-template.yml

```diff
@@ -870,6 +885,10 @@ objects:
             - --kas-fleetshard-addon-id=${KAS_FLEETSHARD_ADDON_ID}
             - --cluster-logging-operator-addon-id=${CLUSTER_LOGGING_OPERATOR_ADDON_ID}
             - --alsologtostderr
+            - --repeat-interval=${REPEAT_INTERVAL}
+            - --leader-election-repeat-interval=${REPEAT_INTERVAL}
+            - --lease-renew-time=${LEASE_RENEW_TIME}
```

Comment:
These flag names may be quite difficult to interpret. So I am open for adjustments :)

Reply:
I'd suggest:
```
--reconciler-repeat-interval
--leader-election-reconciler-repeat-interval
--leader-lease-expiration-time
```

Reply:
Thanks @machi1990